### PR TITLE
add rock 3b accessories

### PR DESCRIPTION
**Pull request by melooooooooooo**
###### Review guide

To review this PR locally, please run the following commands:

```bash
PR= # the current PR number
git fetch origin pull/${PR}/head
git switch --detach FETCH_HEAD
yarn start
yarn start --locale en
# If you want to start 2 locales side-by-side, use the following command on Linux:
# https://github.com/facebook/docusaurus/issues/7377#issuecomment-1167192715
yarn start & DOCUSAURUS_GENERATED_FILES_DIR_NAME=.docusaurus/en yarn start --locale en --port 3001
kill %1
```

### CODE REVIEW

File: docs/Home.md [hunk 1]
```diff
@@ -55,6 +55,50 @@ sidebar_custom_props: { product_docs: [
...
+          Product_Line_zh: "ROCK 3 系列",
+          Product_Line_en: "ROCK 3 Series",
+          # 产品对应的所有文档
+          docs:
+            [
+              {
+                title_zh: "新手入门指南",
+                info_zh: "从这里开始你的 ROCK 3B 之旅",
+                title_en: "Getting Started Guide",
+                info_en: "Start Your ROCK 3A Journey Here",
```
Comment (peterwang2050) [inline docs/Home.md:75]: ROCK 3B

Comment (melooooooooooo) [reply docs/Home.md]: 已经修改

File: docs/Home.md [hunk 2]
```diff
@@ -55,6 +55,50 @@ sidebar_custom_props: { product_docs: [
...
+                title_en: "Compile Source Guide",
+                info_en: "Compile the Source Code",
+                url: "/rock3/lowlevel-dev",
+              },
+              {
+                title_zh: "官方系统镜像",
+                info_zh: "支持多种镜像下载",
+                title_en: "Image Downloads",
+                info_en: "Support Multiple Image Downloads",
+                url: "/rock3/images",
```
Comment (peterwang2050) [inline docs/Home.md:90]: Image Download
Support Multiple Images Download

File: docs/Home.md [hunk 3]
```diff
@@ -98,15 +98,15 @@ sidebar_custom_props: { product_docs: [
               {
                 title_zh: "官方系统镜像",
                 info_zh: "支持多种镜像下载",
-                title_en: "Image Downloads",
-                info_en: "Support Multiple Image Downloads",
+                title_en: "Image Download",
+                info_en: "Support Multiple Image Download",
                 url: "/rock3/images",
```
Comment (peterwang2050) [inline docs/Home.md:103]: Multiple Image => Multiple Images

File: i18n/en/docusaurus-plugin-content-docs/current/rock3/rock3b/accessories/25w-poe-hat.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+Please refer to [Device Tree Configuration](/radxa-os/rsetup/devicetree#how-to-enable-an-overlay) to enable the overlay named "Enable Radxa 25W PoE HAT".
+
+```bash.
+  [*] Enable Radxa 25W PoE HAT
+```
+
+Finally, reboot the device to load the Overlay.
+
+:::info
+If you cannot find the corresponding Overlay in `rsetup`, please refer to [System Update Guide](/radxa-os/rsetup#system-update) to first perform a system update and then perform the above.
```
Comment (peterwang2050) [inline i18n/en/docusaurus-plugin-content-docs/current/rock3/rock3b/accessories/25w-poe-hat.md:33]: to update the system first and then do the steps above

File: i18n/en/docusaurus-plugin-content-docs/current/rock3/rock3b/accessories/25w-poe-hat.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+
+## Installation Procedure
+
+Please refer to [Radxa 25W PoE HAT](/i18n/en/docusaurus-plugin-content-docs/current/accessories/radxa-25w-poe.md)
+
+## Fan Speed Adjustment
+
+### Enable 25W PoE Overlay
+
+If you need to enable the temperature control fan and temperature sensor, you need to enable the corresponding Overlay.
```
Comment (peterwang2050) [inline i18n/en/docusaurus-plugin-content-docs/current/rock3/rock3b/accessories/25w-poe-hat.md:22]: enable the fan and temperature sensor

File: i18n/en/docusaurus-plugin-content-docs/current/rock3/rock3b/accessories/25w-poe-hat.md [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+
+Please refer to [Device Tree Configuration](/radxa-os/rsetup/devicetree#how-to-enable-an-overlay) to enable the overlay named "Enable Radxa 25W PoE HAT".
+
+```bash.
+  [*] Enable Radxa 25W PoE HAT
+```
+
+Finally, reboot the device to load the Overlay.
+
```
Comment (peterwang2050) [inline i18n/en/docusaurus-plugin-content-docs/current/rock3/rock3b/accessories/25w-poe-hat.md:31]: reboot the device to enable the overlay

File: i18n/en/docusaurus-plugin-content-docs/current/rock3/rock3b/accessories/25w-poe-hat.md [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+:::info
+If you cannot find the corresponding Overlay in `rsetup`, please refer to [System Update Guide](/radxa-os/rsetup#system-update) to first perform a system update and then perform the above.
+:::
+
+5. Checking Temperature Sensor Readings
+
```
Comment (peterwang2050) [inline i18n/en/docusaurus-plugin-content-docs/current/rock3/rock3b/accessories/25w-poe-hat.md:37]: Reading data from Temperature Sensor

File: i18n/en/docusaurus-plugin-content-docs/current/rock3/rock3b/accessories/_category_.json [hunk 8]
```diff
@@ -0,0 +1,8 @@
+{
+  "label": "Accessories Guides",
+  "link": {
```
Comment (peterwang2050) [inline i18n/en/docusaurus-plugin-content-docs/current/rock3/rock3b/accessories/_category_.json:3]: Guides => Guide

File: i18n/en/docusaurus-plugin-content-docs/current/rock3/rock3b/accessories/heatsink-4012.md [hunk 9]
```diff
@@ -0,0 +1,139 @@
...
+## List of Accessories
+
+- Heatsink 4012
+- Thermal Conductive Silicone Gel
+- Thermal Conductive Silicone Grease
+
+## Installation Procedure
+
+- Heatsink 4012 is a heatsink fan that can be used with the ROCK 3B. The installation instructions are shown in the diagram below:
+
```
Comment (peterwang2050) [inline i18n/en/docusaurus-plugin-content-docs/current/rock3/rock3b/accessories/heatsink-4012.md:17]: The installation instruction is shown below

File: i18n/en/docusaurus-plugin-content-docs/current/rock3/rock3b/accessories/heatsink-4012.md [hunk 10]
```diff
@@ -0,0 +1,139 @@
...
+- Align the buckle with the mounting holes on the ROCK 5B and press it down.
+- Connect the power cable to the fan socket.
+
+![Heatsink 4012](/img/rock3/3b/rock_3b_with_heatsink.webp)
+
+## Configuration
+
+The operating system has three modes by default：
+
+- **power_allocator**: The system defaults to fanless mode or DC fan mode. Make sure that the machine can still work stably without a cooling fan;
```
Comment (peterwang2050) [inline i18n/en/docusaurus-plugin-content-docs/current/rock3/rock3b/accessories/heatsink-4012.md:35]: The system is in the fanless mode or DC fan mode  by default

File: i18n/en/docusaurus-plugin-content-docs/current/rock3/rock3b/accessories/heatsink-4012.md [hunk 11]
```diff
@@ -0,0 +1,139 @@
...
+- **user_space**: Manually control cooling fan mode. Users can control the speed of the cooling fan through the command terminal according to your needs;
```
Comment (peterwang2050) [inline i18n/en/docusaurus-plugin-content-docs/current/rock3/rock3b/accessories/heatsink-4012.md:36]: Mode of manual control of the fan, User can

File: i18n/en/docusaurus-plugin-content-docs/current/rock3/rock3b/accessories/heatsink-4012.md [hunk 12]
```diff
@@ -0,0 +1,139 @@
...
+- **step_wise**: automatic temperature adjustment mode. When the temperature of the CPU is below 60°C, the cooling fan is in a dormant state; And when the temperature of the CPU reaches above 60°C, the cooling fan starts to work.
```
Comment (peterwang2050) [inline i18n/en/docusaurus-plugin-content-docs/current/rock3/rock3b/accessories/heatsink-4012.md:37]: automatic ..... mode => Mode of  Automatic temperature adjustment

File: i18n/en/docusaurus-plugin-content-docs/current/rock3/rock3b/accessories/heatsink-4012.md [hunk 13]
```diff
@@ -0,0 +1,139 @@
...
+  **Note: When ROCK 5B is in shutdown or sleep state, the cooling fan does not work.**
```
Comment (peterwang2050) [inline i18n/en/docusaurus-plugin-content-docs/current/rock3/rock3b/accessories/heatsink-4012.md:38]: ROCK 5B => ROCK 3B ?

File: i18n/en/docusaurus-plugin-content-docs/current/rock3/rock3b/accessories/heatsink-4012.md [hunk 14]
```diff
@@ -0,0 +1,139 @@
...
+│                                                                                                  │
+│                                                                                                  │
+│                                                                                                  │
+│                                                                                                  │
+│                                 <ok>                        <Cancel>                             │
+└──────────────────────────────────────────────────────────────────────────────────────────────────│
+```
+
+If you choose `user_space` mode, you need to manually control the cooling fan.
+
```
Comment (peterwang2050) [inline i18n/en/docusaurus-plugin-content-docs/current/rock3/rock3b/accessories/heatsink-4012.md:100]: control the fan manually, 不用每次都写cooling fan ,大家知道的

File: i18n/en/docusaurus-plugin-content-docs/current/rock3/rock3b/accessories/heatsink-4012.md [hunk 15]
```diff
@@ -0,0 +1,139 @@
...
+```
+radxa@rock-3b: cat /sys/class/thermal/cooling_device1/type
+pwm-fan
+```
+
+**Note: The following operations take `cooling_device1` as an example.**
+
+Directly open the highest speed:
+
```
Comment (peterwang2050) [inline i18n/en/docusaurus-plugin-content-docs/current/rock3/rock3b/accessories/heatsink-4012.md:117]: Set the  maximum  speed directly

File: i18n/en/docusaurus-plugin-content-docs/current/rock3/rock3b/accessories/heatsink-4012.md [hunk 16]
```diff
@@ -0,0 +1,139 @@
...
+```
+radxa@rock-3b:~$ sudo cp /sys/class/thermal/cooling_device1/max_state /sys/class/thermal/cooling_device1/cur_state
+```
+
+You can check how many speeds the cooling fan supports through the following:
+
```
Comment (peterwang2050) [inline i18n/en/docusaurus-plugin-content-docs/current/rock3/rock3b/accessories/heatsink-4012.md:123]: by the following command

File: i18n/en/docusaurus-plugin-content-docs/current/rock3/rock3b/accessories/heatsink-4012.md [hunk 17]
```diff
@@ -0,0 +1,139 @@
...
+```
+radxa@rock-3b:~$ cat /sys/class/thermal/cooling_device1/max_state
+4
+```
+
+If you want to select the third speed, follow the instructions below:
```
Comment (peterwang2050) [inline i18n/en/docusaurus-plugin-content-docs/current/rock3/rock3b/accessories/heatsink-4012.md:129]: If you want to select level 3, follow the instruction below:

**Review by peterwang2050 — APPROVED** [review]

File: docs/rock3/rock3b/accessories/heatsink-4012.md [hunk 18]
```diff
@@ -0,0 +1,138 @@
...
+
+- 将散热硅胶贴在 Heatsink 4012 上，露出 SoC 位置用于涂上导热硅脂；
+
+![Heatsink 4012](/img/rock5b/heatsink4012-use-1.webp)
+
+- 把散热硅胶涂在 SoC 芯片上；
+
+![Heatsink 4012](/img/rock5b/heatsink4012-use-2.webp)
+
+- 将卡扣对准 ROCK 5B 的孔位按压下去；
```
Comment (peterwang2050) [inline docs/rock3/rock3b/accessories/heatsink-4012.md:26]: ROCK 5B ?

File: i18n/en/docusaurus-plugin-content-docs/current/rock3/rock3b/accessories/heatsink-4012.md [hunk 19]
```diff
@@ -0,0 +1,139 @@
...
+- Apply thermal silicone adhesive to the Heatsink 4012, exposing the SoC position for the application of thermal silicone grease.
+
+![Heatsink 4012](/img/rock5b/heatsink4012-use-1.webp)
+
+- Apply thermal silicone adhesive to the SoC chip.
+
+![Heatsink 4012](/img/rock5b/heatsink4012-use-2.webp)
+
+- Align the buckle with the mounting holes on the ROCK 5B and press it down.
+- Connect the power cable to the fan socket.
```
Comment (peterwang2050) [inline i18n/en/docusaurus-plugin-content-docs/current/rock3/rock3b/accessories/heatsink-4012.md:27]: 5B ?

**Review by peterwang2050 — APPROVED** [review]

**Pull request merged**